smart_card_connector_app/src/window.css

```diff
@@ -175,3 +175,34 @@ body {
   top: -2px;
   width: 20px;
 }
+
+.tool-tip {
+  background: rgb(30, 30, 30);
+  border-radius: 1px;
+  color: white;
+  display: inline-block;
+  font-size: 12px;
+  height: 15px;
+  padding: 2px 5px 0px 5px;
+  position: absolute;
+  top: 150%;
+  text-align: center;
+  visibility: hidden;
+  width: max-content;
+}
+
+.tool-tip-init::after {
+  content: "Initializing card...";
```

Comment:
I am not a fan of having the content set by CSS - all content should come from HTML/JS side. CSS should only be concerned with styling the content but not with any concrete strings, just as how HTML should not use any style tags. 
Otherwise, the line between HTML and CSS becomes blurry, and with this approach, i18n also becomes impossible.
Just put the strings that should be displayed as part of the element that you create with JS.

Reply:
Makes sense. Thanks for that!